### DISCUSSION TITLE
[灌水]各位绳匠，剧情打不动了怎么办，这都什么乱七八糟的

### DISCUSSION THREAD

**Discussion by JIIIMXu** [share121/inter-knot#2614] [其他]
![68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f37306235623063393534336565353038306336616539326664323739373839322e706e67](https://github.com/user-attachments/assets/907e4b33-9133-4b48-a80d-8f348302719f)

41级了，不想打剧情，感觉剧情写的依托，而且走格子真的痛苦，有没有和我一个想法的

**Comment by HKLHaoBin** [comment]
没关系 如果感觉不合适 就退游吧

**Comment by Sunny-Ding110** [comment]
剧情确实比较直白，就是说玩不下去可以退游的（认真）

**Comment by wwwsheepwww** [comment]
鼓励玩的不开心就退游，或者等等看有没有优化

**Comment by JIIIMXu** [comment]
想退游，但是充钱了（

**Comment by ElvenShark** [comment]
玩不动就退呗，别折磨自己，游戏是取乐的，不过我个人感觉挺好玩的，每个人不一样

**Comment by SWYH34827** [comment]
玩不下去就退游，就这么简单

**Comment by lixiao189** [comment]
等上述痛点改掉了再回坑就行，反正米哈游又不会倒闭
